core/templates/dev/head/pages/profile/profile.html

```diff
@@ -134,12 +134,13 @@ <h2 class="oppia-profile-username-small-screen">
       </md-content>
 
       <span class="oppia-profile-portfolio-pages" ng-if="numUserPortfolioExplorations > 6">
-        <i class="material-icons md-18" ng-disabled="currentPageNumber == 0"
+        <i ng-class="{'material-icons md-18 oppia-navigation-button':true,
+                      'disabled':startingExplorationNumber==1}"
            ng-click="goToPreviousPage()">
```

Comment:
Instead of using ng-class with a dict, I still think ng-disabled usage is cleaner here. Can we do that instead? Ditto below.

Reply:
Sure.!
done.

Reply:
do ng-disabled works for i tag? (Like it works for form elements button, input etc. but doesn't work for the anchor tag, so not sure whether it works for this or not.)
I would suggest to do a manual test.

Reply:
do ng-disabled works for i tag? (Like it works for form elements like button, input etc. but doesn't work for the anchor tag, so not sure whether it works for this or not.)
I would suggest doing a manual test.

Reply:
yes.! you are right. @DubeySandeep 
even when it is disabled we can click on it. 
but,it was added just to give some extra styling ( light color background).